docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,336 @@
+c) And *how* are you going to implement the proposed solution?
+
+Depending on the type of your blueprint, the answer for the last question (c)
+could be written in pseudocode, general text or might even not be necessary
+(although desired) — more details on the section named Specification.
+
+When using RFCs as email threads, there are no sections or headers, each
+contributor will try to send the RFC without following formal sections and
+headers. RFCs, as we use today are just thread discussions and are not focused
+on future review/reading.
+
+  * Make sure that another peer will be able to implement a feature/requirement.
+
+Blueprints are not for you; they are for the community. If you know that a
+problem exists and know how to fix it, the most natural course of action would
+be to start coding this fix and submitting a Pull Request. While this is still
+valid for most of the cases, some important architectural changes must be
+discussed first to explain the “why”, “what” and “how” to keep everyone on the
+same page, avoid unnecessary coding, and most importantly: allow others to
+implement it in case you are not available.
+
+  * Improve the code review quality and time.
+
+Having a better understanding of the problem and the big picture is better for
+code review. It is harder to capture that from the Pull Request (PR)
+description and PR changes. Developers that are aware of the problem tend to
+review your changes with the problem in mind and hence more quickly. 
+
+  * Reduce onboarding time for new members.
+
+Having a history of how we made an architectural decision and why we are
+implementing it this way will give new members reading material to understand
+our project, avoiding unnecessary discussions and meetings to explain
+something.
+
+  * Create a common standard that will make it easier for readers.
+
+With an open RFC, authors tend to organize the ideas in different ways with
+different sections. Having a very well structured document with common sections
+will make it easier for readers to understand the problem and the solution.
+
+  * Track the progress of a significant implementation.
+
+We could use the blueprints header “status” line to track the progress of some
+features. We could even have a page parsing and listing all the blueprints with
+the title, author, status, and target milestone for that feature.
+
+  * Find the middle ground between “overthinking” and “auto-pilot.”
+
+Last but not least: we are *not trying* to overthink here and/or slow down our
+development processes. The idea is to have a lightweight document, very
+objective that will save us time in the medium and long run. We don’t have to
+overthink by trying to handle any possible scenario outside of ones we actually
+have a use case for. But we should also avoid the “auto-pilot” mode in our
+contributions where we fixing issues as quick as possible without thinking on
+the big picture, it is not healthy for the project.
```

Comment:
I guess this also implies we don't have any requirements about the size of the blueprints and a blueprint might be fairly small and to the point, correct?

Reply:
Yes.

Reply:
Might be worth mentioning it but I guess it is clear to everyone.